x/staking/client/cli/query.go

```diff
@@ -198,30 +194,25 @@ $ %s query staking redelegations-from cosmosvaloper1gghjut3ccd8ay0zduzj64hwre2fx
 		),
 		Args: cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			clientCtx := client.NewContext().WithCodec(cdc).WithJSONMarshaler(cdc)
+			clientCtx := client.GetClientContextFromCmd(cmd)
+			queryClient := types.NewQueryClient(clientCtx)
 
 			valSrcAddr, err := sdk.ValAddressFromBech32(args[0])
 			if err != nil {
 				return err
 			}
 
-			bz, err := cdc.MarshalJSON(types.QueryRedelegationParams{SrcValidatorAddr: valSrcAddr})
-			if err != nil {
-				return err
+			params := &types.QueryRedelegationsRequest{
+				SrcValidatorAddr: valSrcAddr,
+				Req:              &query.PageRequest{},
```

Comment:
should we add the flags for the page request? cc @clevinson @aaronc

Reply:
Yes, I think @atheeshp is addressing the required flags in another PR. It is missing in other modules too

Reply:
Or may be we can address the flags in this PR itself (also for already merged PRs) so that future PRs can use the same.

Reply:
We probably can put this in a helper like `ReadPageRequest(*FlagSet) *PageRequest`